qa-tests-backend/src/main/groovy/services/ImageIntegrationService.groovy

```diff
@@ -158,6 +158,11 @@ class ImageIntegrationService extends BaseService {
     }
 
     static String addStackroxScannerIntegration() {
+        ImageIntegrationOuterClass.ImageIntegration existing =
+            getImageIntegrationByName(StackroxScannerIntegration.name())
+        if (existing) {
+            return existing.id
+        }
         return StackroxScannerIntegration.createDefaultIntegration()
```

Comment:
This assumes that `StackroxScannerIntegration.createDefaultIntegration()` creates an integration with a name that `StackroxScannerIntegration.name()` returns. I feel like making that assumption somewhere inside `StackroxScannerIntegration` would be safer than making it here at. How about instead introducing a`StackroxScannerIntegration.createDefaultIntegrationIfMissing()` method that would contain the logic you're adding here, and calling it in this method?